docker-compose.yml

```diff
@@ -0,0 +1,59 @@
+version: '3.8'
+
+volumes:
```

Comment:
volumes are required for data persistence, plus you will not ship your actual pc, hence the bind between the volume and the directory, plus :
> volumes create extra directories that are a bridge between the machine and the virtual container
....... that's exactly what is needed, unless you want to rebuild your image every single time and have to wait 30 mins for the server to boot up instead of hot reaload (and no, you can ask echo, it took him arround 40 mins to build/start yesterday at building's pc).

Reply:
Ok, if you think its better. I just thought bind mounts were interesting also. Let's use volumes.